Accepted-RFCs/RFC-012-internal-knowledge-base.md

```diff
@@ -0,0 +1,121 @@
+
+Ideally, our docs would make use of our templates—in fact, they would make for great use/test cases. However, we would allow for a "quickly written" status, in cases where information needs to be dumped from someone's mind, but the time/effort cost in polishing isn't currently possible.
+
+We'd have a mechanism for checking the freshness of content, though we can defer the details of that until later.
+
+### GitLab wiki: a.k.a. Do we already have this?
+There's a wiki in GitLab, and [Aaron has release weight information there](https://gitlab.com/tgdp/governance/-/wikis/Guide-to-assigning-weight-scores-to-issues-and-epics-%28release-planning%29). Should we just go with that? Does that make sense for us as a group of groups?
+
+Is the wiki going to be oversaturated with other purposes? Would it be difficult to use search functionality when looking for KB articles, due to non-KB stuff in the wiki also populating the search results?
+
+## Consequences
+
+We'd have a central place for information.
+
+Said information though could become out of date without a contributor realizing it, due to the nature of knowledge bases and the bandwidth of contributors.
+
+We also need to decide on the access control. Having new contributors being able to edit can be great for finding holes in the content, but could also be an issue if contributors put misunderstandings into our KB without others noticing right away.
+
+This space would also be great for small procedural elements that should be documented, but don't need a full RFC for said documentation (such as the recent "break/restart" process the co-chairs initiated). That way such info isn't lost to the ether due to a lack of documentation procedure for such things.
+
+
+## Links and prior art
+
+{This section is optional if you want to [link](https://example.com) to other resources.}
+
+
+## Open questions
+
+{This section is optional and is where you can list questions that won't be resolved by this RFC, including those raised in comments from community members.}
+
+
+## Decisions deferred
+
+* Freshness mechanisms
+* Access control (a.k.a. who can edit/update)
+
+
+## Feedback
+
+{If you accept feedback from a community member, you will incorporate it into your RFC before it is accepted.
+If you reject feedback, note that rejected feedback here before resolving the conversation.}
+
+
+## Organizational dependencies
+
+* The tech team would be involved in implementation
+* Content strategy may have thoughts, even with this being internal-facing info
+
+## Implementation checklist
+
+If this proposal is accepted, the following tasks must be completed:
+
+- [ ] Set the site/wiki/whatever up
+- [ ] Draft the core info on how to use & contribute, so we're all on the same page about it
+- [ ] Create the initial structure/document tree
+- [ ] Ask group leads to dump critical or infrastructure knowledge from their brains or disparate notes into the KB
+
```

Comment:
- [ ] Scrape the existing GitHub repos and wikis for any project and process information that is relevant to capture in the  KB
 - [ ] Comb existing Google drives per working group for relevant info to move into the KB

Reply:
Added